index.html

```diff
@@ -237,5 +237,22 @@ <h4>Skills</h4>
           </div>  
         </tag>
     </section>
+     <section class="contact-container">
+        <div class="contact-me">
+          <h2 class="contact-heading">Contact me</h2>
+          <p class="contact-text">If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I'd love to help with it.
+           </p>
+        <div class="form">
+          <form action="submit" class="contact-form">
+            <input class="contact-input"  value="Leslie" type="text" >
+            <input class="contact-input" type="email" value="leeegd99@gmail.com">
+            <textarea class="contact-input" cols="30" rows="9"
+                placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Kindly use `required` and `maxlength` attributes on this element to trigger the validations discussed above. We would like to prevent the form from getting submitted if someone inputs a text which is more than 500 characters long. 🥸️